src/App.tsx

```diff
@@ -1,61 +1,131 @@
 import './App.scss';
+import { FormEvent, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+const copyOfUsersFromServer: User[] = [...usersFromServer];
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+const biggestTodoId = () => {
+  const ids = [...todos].map(todo => {
```

Comment:
If you are using a non-mutating array method, you don't need to create a copy of the array